eth/tracers/logger/logger.go

```diff
@@ -216,12 +229,33 @@ func (l *StructLogger) CaptureEnd(output []byte, gasUsed uint64, t time.Duration
 			fmt.Printf(" error: %v\n", err)
 		}
 	}
+	// TODO: duplicate info, re-use existing vars
+	l.result = &core.ExecutionResult{
+		UsedGas:    gasUsed + l.intrinsicGas,
```

Comment:
So it's a breaking change? Although I know it's a long existent issue, but have no idea what's the spec for it.

Feel free to ignore my comment.

Reply:
Gary I'm not sure I remember what your comment was referring to. What do you think is a breaking change?

Reply:
Previously the `UsedGas` refers to the usage in evm. While now the additional `intrinsicGas` is added.
I would say it's right and fix the long existent issue, but the tracing result will be different though.

Reply:
So you're right that the gas usage is not accurate here. But the problem is that this new code doesn't take refunds into account. Also that the intrinsic cost that we calculate here is not valid for txes which come with access lists, but the access list is not exposed to tracers so we can't really reliably calculate that.

I think the issue you're referring to was that JS tracers didn't consider `intrinsicGas` which was fixed here: https://github.com/ethereum/go-ethereum/pull/22038/files#diff-032a64470618242b2ba799c3e7f4c2d86c5a755b8e5bf36e10e84053dd0dbaa4R563 (note that's also not accurate anymore because of access lists)